Makefile

```diff
@@ -34,7 +34,7 @@ lint: ## Verifies `golint` passes.
 .PHONY: test
 test: ## Runs the go tests.
 	@echo "+ $@"
-	$(V)GO111MODULE=on $(GO) test -v -tags "$(BUILDTAGS) cgo" $(shell $(GO) list ./... | grep -v vendor)
+	$(V)GO111MODULE=on $(GO) test -v -tags "$(BUILDTAGS) cgo" ./...
```

Comment:
This is good, but `fmt` and `lint` targets still have `grep -v '.pb.go:' | grep -v vendor`. It doesn't look like there are any `.pb.go` files, and `vendor` isn't being used any more. Can we simplify that?

Reply:
sure, why not!